.github/workflows/autoreplies/pypi_utils.py

```diff
@@ -0,0 +1,91 @@
+import re
+import time
+from typing import Dict
+
+import requests
+from bs4 import BeautifulSoup
+
+
+def get_packages_by_user(username: str) -> list:
+    """Parse html to get a list of packages for a given PyPI user.
+
+    The pypi api does not provide a way to get a list of packages for a user, hence crawling the html.
```

Comment:
Unless I'm wrong there is no api I can use to directly query a pypi user to determine the packages they maintain. Manually parsing the html to find it is not the most stable solution, but is working for now.